src/main/java/salesforce/utils/EnvironmentInitialConfig.java

```
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package salesforce.utils;
+
+import salesforce.config.EnvironmentConfig;
+import java.io.FileWriter;
+import java.lang.reflect.Field;
+
+public class EnvironmentInitialConfig {
+
+    public EnvironmentInitialConfig() { }
+
+    /**
+     * Creates a .env file with required fields.
+     */
+    public void createEnvFile() {
+        try {
+            FileWriter envOutput = new FileWriter(".env");
+            envOutput.write(setVariables());
+            envOutput.close();
+        } catch (Exception e) {
+            e.printStackTrace();
```

You should throw the exception.